plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -57,7 +63,33 @@ test.describe.serial( 'Add New Simple Product Page', () => {
 		} );
 		await page.locator( '#title' ).fill( virtualProductName );
 		await page.locator( '#_regular_price' ).fill( productPrice );
+		await page.locator( '#_sale_price' ).fill( salePrice );
 		await page.locator( '#_virtual' ).click();
+
+		// Fill in a product description
+		await page
+			.frameLocator( '#content_ifr' )
+			.locator( '.wp-editor' )
+			.fill( productDescription );
```

Comment:
The locator also didn't work for me when the "Product description" field was set to "Text". Would be great to add a step to click on the "Visual" tab first to ensure robustness of the product description locator. 

Alternatively, you may also use Playwright Locator's [or](https://playwright.dev/docs/api/class-locator#locator-or) command so that you will end up `or`ing two locators for the Product description textbox, and these `or`ed locators will work on both "Visual" and "Text" mode.

I think either of those 2 approaches would work.

Reply:
Nice one, thanks for suggesting to click Visual beforehand.